lambda/save-raw-events/save-raw-events.ts

```diff
@@ -0,0 +1,92 @@
+import { SQSEvent } from "aws-lambda";
+import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
+import {
+  DynamoDBDocumentClient,
+  PutCommand,
+  PutCommandOutput,
+} from "@aws-sdk/lib-dynamodb";
+import {
+  SendMessageCommand,
+  SendMessageRequest,
+  SQSClient,
+} from "@aws-sdk/client-sqs";
+import { TxmaEvent, UserData } from "./models";
+
+const { TABLE_NAME } = process.env;
```

Comment:
These could be collapsed with L27.

But! I'd suggest moving the this line to L64 so it sits within the `writeRawTxmaEvent` function,
And L27 down to the handler function so it's on something like L76.

There's some occasional test flake that I was getting, ([see here](https://github.com/alphagov/di-account-management-backend/pull/46)), best I can tell it's something to do with the order the tests are run in, and the env occasionally being null.

I wondered if it's to do with the way that jest brings in a function vs a file?

In any case, these two aren't widely re-used so lets put them into the functions as a specific concern of that function